packages/scripts/package.json

```diff
@@ -45,33 +47,53 @@
     "enzyme-to-json": "^3.0.0",
     "eslint": "^4.0.0",
     "eslint-config-airbnb": "^11.1.0",
+    "eslint-plugin-angular": "^1.4.1",
     "eslint-plugin-import": "^1.16.0",
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.3.0",
     "file-loader": "^2.0.0",
+    "html-loader": "^0.5.5",
     "html-webpack-plugin": "3.2.0",
+    "jasmine": "2.5.2",
+    "jasmine-core": "2.5.2",
     "jest": "^24.0.0",
     "jest-environment-jsdom": "^23.4.0",
     "jest-environment-jsdom-global": "^1.1.1",
+    "karma": "^1.4.1",
+    "karma-coverage": "^1.1.1",
+    "karma-jasmine": "^1.1.0",
+    "karma-jasmine-diff-reporter": "^1.2.0",
+    "karma-junit-reporter": "^1.2.0",
+    "karma-phantomjs-launcher": "^1.0.2",
+    "karma-webpack": "^4.0.0-rc.6",
     "lodash.get": "^4.4.2",
     "lodash.mergewith": "^4.6.1",
     "mini-css-extract-plugin": "^0.4.0",
+    "ng-annotate-loader": "^0.6.1",
+    "ng-annotate-patched": "^1.9.0",
+    "ngtemplate-loader": "^2.0.1",
     "node-sass": "^4.7.2",
-    "optimize-css-assets-webpack-plugin": "4.0.0",
+    "null-loader": "^0.1.1",
+    "optimize-css-assets-webpack-plugin": "^5.0.1",
+    "phantomjs-prebuilt": "^2.1.7",
     "postcss-loader": "^3.0.0",
+    "postcss-safe-parser": "^4.0.1",
     "raf": "^3.4.0",
     "react-test-renderer": "^16.8.0",
     "resolve-url-loader": "2.3.0",
     "rimraf": "^2.6.2",
     "sass-loader": "^7.1.0",
+    "style-loader": "^0.23.1",
+    "terser-webpack-plugin": "^1.2.3",
     "uglifyjs-webpack-plugin": "1.2.7",
     "url-loader": "^1.1.2",
     "webpack": "^4.19.0",
     "webpack-bundle-analyzer": "^3.0.4",
     "webpack-cli": "^3.1.0",
     "webpack-dev-server": "^3.1.11",
     "whatwg-fetch": "^2.0.3",
-    "which": "^1.3.0"
+    "which": "^1.3.0",
+    "yargs": "^13.2.1"
```

Comment:
not usefull, please remove

Reply:
In fact it is used in `karma.conf.js` to set the single run arg

Reply:
flag could not be passed from @talend/scripts...